gulpfile.js

```diff
@@ -71,13 +111,19 @@ gulp.task('browserify', ['browserify-bolt',
                          'browserify-generator-test',
                          'browserify-mail-test']);
 
+
+
 // Runs the Mocha test suite
 gulp.task('test', ['build'], function() {
   return gulp.src(TEST_FILES)
     .pipe(mocha({ui: 'tdd'}));
 });
 
-gulp.task('default', ['lint', 'build', 'test']);
+gulp.task('default', ['build', 'test']);
+
+gulp.task('watch', ['default'], function() {
+    gulp.watch(['lib/*.ts', 'test/*.ts'], ['default']);
+});
 
```

Comment:
Yeah I just did an in-place conversion to typescript, I think moving .ts files to src makes more sense